### DISCUSSION TITLE
Refresh thumbnail

### DISCUSSION THREAD

**Discussion by mister-teddy** [dailydotdev/daily#1040] [General]
Hi guys,
I wonder how long it would take for my new blog thumbnail to appear on daily.dev?
P/S: here is my [blog](https://app.daily.dev/posts/qefVamSZX). As you can see, the thumbnail is quite outdated compared to the one on dev.to 😅

**Comment by idoshamun** [comment]
Hi @teddyfullstack,

We don't refresh thumbnails once we fetch the article from the source.
Thumbnails are mostly immutable.
